release/promote.sh

```diff
@@ -25,9 +25,10 @@ function pull_tag_push() {
 }
 
 if [ "$PLATFORM" = "docker" ]; then
-  PUBLISH_HALYARD_DOCKER_IMAGE_BASE=${PUBLISH_HALYARD_DOCKER_IMAGE_BASE:-gcr.io/spinnaker-marketplace/halyard}
-  SOURCE_IMAGE=$PUBLISH_HALYARD_DOCKER_IMAGE_BASE:$SOURCE_VERSION
-  TARGET_IMAGE=$PUBLISH_HALYARD_DOCKER_IMAGE_BASE:$TARGET_VERSION
+  PUBLISH_HALYARD_DOCKER_IMAGE_SRC_BASE=${PUBLISH_HALYARD_DOCKER_IMAGE_SRC_BASE:-us-docker.pkg.dev/spinnaker-community/nightly/halyard}
+  PUBLISH_HALYARD_DOCKER_IMAGE_TARGET_BASE=${PUBLISH_HALYARD_DOCKER_IMAGE_TARGET_BASE:-us-docker.pkg.dev/spinnaker-community/releases/halyard}
+  SOURCE_IMAGE=PUBLISH_HALYARD_DOCKER_IMAGE_SRC_BASE:$SOURCE_VERSION
+  TARGET_IMAGE=PUBLISH_HALYARD_DOCKER_IMAGE_TARGET_BASE:$TARGET_VERSION
```

Comment:
The `build_halyard` buildtool command also calls this script to publish a `nightly` version, so I don't think this will work...

Reply:
I know, I haven't gotten that far but will change that before merging this